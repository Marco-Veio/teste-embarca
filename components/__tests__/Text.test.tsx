import React from "react";
import renderer from "react-test-renderer";

import { Text } from "../Themed";

describe("Text component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Text>Star Wars</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
