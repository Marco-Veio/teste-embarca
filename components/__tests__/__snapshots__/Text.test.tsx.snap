// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      undefined,
    ]
  }
>
  Star Wars
</Text>
`;
